dowhy/causal_identifiers/IDIdentifier.py

```diff
@@ -0,0 +1,260 @@
+from builtins import isinstance
```

Comment:
is this needed? I thought isinstance can be directly called, without any imports.